chef/cookbooks/bind9/recipes/default.rb

```diff
@@ -322,5 +311,14 @@ def make_zone(zone)
   notifies :restart, "service[bind9]", :immediately
```

Comment:
You mean because the `bind9` service declaration appears below it?  Surely Chef's two-phase approach will prevent that?  I think I also tested but TBH I can't remember now.